Library/Formula/agda.rb

```diff
@@ -51,38 +58,61 @@ def install
     cabal_clean_lib
 
     if build.with? "stdlib"
-      resource("stdlib").stage prefix/"agda-stdlib"
+      resource("stdlib").stage lib
```

Comment:
Probably the reason we didn't use `lib` before is everything in there will be linked into `/usr/local/lib`. That's not to say we shouldn't change it but just worth considering.